### DISCUSSION TITLE
JUnit export for tests

### DISCUSSION THREAD

**Discussion by SheatNoisette** [vlang/v#18619] [Ideas]
### Describe the feature

A test runner that exports V tests into a JUnit XML file or similar.

### Use Case

It would allow proper integration into CIs like Github Actions or Gitlab CI.

### Proposed Solution

If you have two files, like `testone_test.v` and `testtwo_test.v` this would generate reports `testone_test_junit.xml` and `testtwo_test_junit.xml` using a flag such as `v -test-runner junit test .`

### Other Information

Any reputable tests export format can be explored, but we should stick to a defacto standard like JUnit XML (Supported by Gitlab CI and Github Action)

### Acknowledgements

- [X] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### Version used

V 0.3.2 18d98a5

### Environment details (OS name and version, etc.)

macOS 12 - Aarch64

**Comment by spytheman** [comment]
~~I only managed to find this: https://xunit.net/docs/format-xml-v2 . Is there a more complete documentation of the expected format somewhere else?~~

Edit: this seems to be better so far: <https://llg.cubic.org/docs/junit/> .

**Comment by JalonSolov** [comment]
https://www.ibm.com/docs/en/developer-for-zos/14.2?topic=formats-junit-xml-format
